_posts/2024-09-30-Boosting-OpenSearch-Development-on-MacOS-with-Apple-Silicon.md

```diff
@@ -1,46 +1,44 @@
 ---
 layout: post
-title:  Boosting OpenSearch Development on MacOS with Apple Silicon
+title:  Streamline your OpenSearch development workflow on macOS with Apple silicon
 authors:
 - zhujiaxiang
-- ssingh
+- sisurab
+- kolchfa
 date: 2024-09-30
```

Comment:
Would need to change this date and the file name to corresponding blog release date once settled.